instrumentation/opentelemetry/init.go

```diff
@@ -237,6 +246,12 @@ func createResources(serviceName string, resources map[string]string) []attribut
 
 // RegisterService creates tracerprovider for a new service and returns a func which can be used to create spans
 func RegisterService(serviceName string, resourceAttributes map[string]string) (sdk.StartSpan, error) {
+	return RegisterServiceWithSpanProcessorWrapper(serviceName, resourceAttributes, nil)
+}
+
+// RegisterService creates tracerprovider for a new service with a wrapper over opentelemetry span processor
```

Comment:
fix comment

Reply:
The wrapper functionality is mentioned in the comment. Let me know if you want me to add/fix something else.

Reply:
`RegisterService` is the wrong function name